src/App.js

```diff
@@ -1,13 +1,40 @@
 import React from 'react';
+import {
+  BrowserRouter as Router, Routes, Route, Link,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
-import './App.css';
+import Home from './components/Home';
+import Quote from './components/Quote';
+import styles from './App.module.css';
 
 class App extends React.PureComponent {
   render() {
     return (
-      <div className="Application">
-        <Calculator />
-      </div>
+      <Router>
+        <div className={styles.Application}>
+          <nav>
+            <h1 className={styles.header}>Math Magicians</h1>
+            <ul className={styles.links}>
+              <li>
+                <Link to="/">Home</Link>
+              </li>
+              |
+              <li>
+                <Link to="/Calculator">Calculator</Link>
+              </li>
+              |
+              <li>
+                <Link to="/Quote">Quote</Link>
+              </li>
+            </ul>
+          </nav>
```

Comment:
[OPTIONAL] I think it would be best if you created a `Navbar.js` component under the `components` directory then render the component inside the `App.js`.

- This will make your REACT adhere to the SOLID principles. This fulfills the `Single functionality` principle in (S)OLID which states that ***a class should have one and only one reason to change, meaning that a class should have only one job.*** In your case, the `Navbar` component is for navigation, and the `App.js` is the main component for Routing/Rendering components.

I hope this makes sense...